packages/dialtone-vue2/recipes/item_layout/contact_info/contact_info.stories.js

```diff
@@ -39,13 +39,7 @@ export const argTypesData = {
   },
 
   avatarIcon: {
-    options: iconsList,
-    control: {
-      type: 'select',
-      labels: {
-        undefined: '(empty)',
-      },
-    },
```

Comment:
Why we're no longer providing the ability to put any icon here? This way is imposible to change it on storybook

Reply:
![image](https://github.com/user-attachments/assets/220a8b65-a3bb-41ec-889f-5a00979b9220)

Reply:
This can still work exactly the same as before I think, you just render a dt-icon within the slot now instead of the prop in the `_default.story.vue`